app/src/app/views/profile-page/profile-page.component.ts

```diff
@@ -36,6 +39,19 @@ export class ProfilePageComponent implements OnInit, OnDestroy {
         this.saveSubscription.unsubscribe();
     }
 
+    /** get list of teams the user is in */
+    private getTeams() {
```

Comment:
We could maybe make a new API route for this so the UI is doing less handling.  Not blocking this being pulled in, just a suggestion.